packages/@webex/plugin-meetings/src/meetings/index.ts

```diff
@@ -827,6 +827,27 @@ export default class Meetings extends WebexPlugin {
     }
   }
 
+  /**
+   * API to toggle stopping ICE Candidates Gathering after first relay candidate,
+   * needs to be called before webex.meetings.joinWithMedia()
+   *
+   * @param {Boolean} newValue
+   * @private
+   * @memberof Meetings
+   * @returns {undefined}
+   */
+  private _toggleStopIceGatheringAfterFirstRelayCandidate(newValue: boolean) {
+    if (typeof newValue !== 'boolean') {
+      return;
+    }
+
+    // @ts-ignore
```

Comment:
```suggestion
    // @ts-expect-error [explanation]
```

expect-error will also show error if for some reason in future below line will start to be correct and potentially stop silencing important errors